roles/auth_proxy/files/contivRule.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+IFS="
+"
+rules_file="/etc/contiv/rules.conf"
+if [ -f "$rules_file" ]
+then
+    while read line; do
+        eval iptables $line
```

Comment:
this is dangerous, if there is anything or anyone changed the file by accident, you may lose root access to the box.
instead i would suggest to put logic in this file about 
1. if rules exists, skip
2. if rule missed, add the rule
And rules can be in different file that templated.
The benefit of doing this are, you don't have to use shell to manipulate iptables in ansible anymore, all you need is to enable and start this service, and you don't have to manage the iptables in serf start up script either, because you can manage the service dependency between these two services.

Reply:
@tiewei , i  understand your concern and would need your help to implement your suggested solution. I am still in learning phase of scripting part :) and very familiar their proper uses in production environment.

Reply:
the eval seems unnecessary, I don't see any reason to eval vs just running `iptables $line`

Reply:
The `eval` is needed because there are spaces in the $var and they will get quoted

Quick example:

```sh
export ARGS="-l -h"
ls $ARGS # fails
eval ls $ARGS # succeeds
```